iocage_lib/ioc_start.py

```diff
@@ -615,6 +614,60 @@ def __start_jail__(self):
             "IOCAGE_NAME": f"ioc-{self.uuid}",
         }
 
+        if nat:
+            # We pass some environment variables to the shell script
+            # for nat based jails currently aiding in doing jail specific
+            # tasks in the host environment
+            pre_start_env = {
+                **os.environ,
+                'INTERNAL_DEFAULT_ROUTER': self.defaultrouter,
+                'INTERNAL_IP': self.ip4_addr.split(
+                    ','
+                )[0].split('|')[-1].split('/')[0]
+            }
+            default_gw_iface = self.host_gateways['ipv4']['interface']
```

Comment:
I imagine self.host_gateways['ipv4']['interface'] will always exist, no matter what?

Reply:
The keys will yes exist no matter what